crates/pathfinder/src/state/sync.rs

```diff
@@ -579,6 +579,24 @@ fn deploy_contract(
         .context("Inserting contract hash into contracts table")
 }
 
+/// Returns the interval to be used when polling the sequencer while at the head of the chain. The
```

Comment:
```suggestion
/// Returns the interval to be used when polling the sequencer or L1 node while at the head of the chain. The
```

Or something like that. docs-wise it might be better to have a short first line, then put the rest of the stuff after a line break.